google-cloud-pubsub/lib/google/cloud/pubsub/project.rb

```diff
@@ -144,10 +182,10 @@ def topic topic_name, project: nil, skip_lookup: nil, async: nil
         #   pubsub = Google::Cloud::Pubsub.new
         #   topic = pubsub.create_topic "my-topic"
         #
```

Comment:
Same as my Subscription comment, I intentionally left examples off since I don't expect most users will want to change the default configuration. But that's also why you're assigned to review. :)